components/announcement_bar/announcement_bar.jsx

```diff
@@ -119,3 +124,20 @@ export default class AnnouncementBar extends React.PureComponent {
         );
     }
 }
+
+function mapStateToProps(state) {
+    return {
+        announcementBarCount: getAnnouncementBarCount(state)
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+        actions: bindActionCreators({
+            incrementAnnouncementBarCount,
+            decrementAnnouncementBarCount,
+        }, dispatch),
+    };
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(AnnouncementBar);
```

Comment:
The connector for this component should go in its own file. You'll have to move this into its own folder for that though.